components/identifier-generator/front/src/feature/tabs/structure/preview/AutoNumberPreview.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {AutoNumber} from '../../../models';
+import {Preview} from 'akeneo-design-system';
+
+type AutoNumberPreviewProps = {
+  property: AutoNumber;
+};
+
+const AutoNumberPreview: React.FC<AutoNumberPreviewProps> = ({property}) => {
+  const {digitsMin, numberMin} = property;
+
+  const getFormattedNumber = (i: number) => {
+    return String(Math.max(i, numberMin)).padStart(digitsMin, '0');
+  };
```

Comment:
If `123` does not change
```suggestion
  const formattedNumber = useMemo(() => String(Math.max(i, numberMin)).padStart(digitsMin, '0');
  , [numberMin, digitsMin]);
```